### DISCUSSION TITLE
Bride of the Embalmer

### DISCUSSION THREAD

**Discussion by TooTall54** [chromiecraft/chromiecraft#2561] [General]
I am unable to get Bride of the Embalmer to work.  I click on the grave but nothing happens.   Eliza does not come out of the grave so I am unable to get the heart.  Any thoughts?

**Comment by anthonygauthier** [comment]
This seems odd to me, I was able to do the quest without any issue last week. Have you tried several times, at different times of the day? An idea would be to try it at server restart (which is at 12:30 AM EST).

**Reply by Shineslippers** [reply]
Answered there https://github.com/chromiecraft/chromiecraft/issues/2503#issuecomment-992352048
